src/polkadot/augment-api-query.ts

```diff
@@ -114,7 +114,7 @@ import type {
   PolymeshPrimitivesTicker,
   PolymeshPrimitivesTransferComplianceAssetTransferCompliance,
   PolymeshPrimitivesTransferComplianceTransferConditionExemptKey,
-  PolymeshRuntimeCiRuntimeSessionKeys,
+  PolymeshRuntimeDevelopRuntimeSessionKeys,
```

Comment:
I've been using the pm2 script in the Polymesh repo since polymesh-local has been flaky.

Its usually in "ci" mode, but I switched it to dev. I don't think these matter too much. going to push to clean up the diff here though